src/ws/cockpit-desktop.in

```diff
@@ -165,23 +165,32 @@ esac
 
 detect_browser
 
+if test -f /.flatpak-info; then
+    ssh_cmd='flatpak-spawn --host ssh'
+    unshare_shell='/bin/bash -c'
+else
+    ssh_cmd='ssh'
+    unshare_shell='unshare --user --map-root-user --net /bin/bash -c'
+fi
+
 # start the bridge; this needs to run in the normal user session/namespace
-coproc ${2:+ssh "$2"} cockpit-bridge
+coproc ${2:+$ssh_cmd "$2"} cockpit-bridge
 trap "kill $COPROC_PID; wait $COPROC_PID || true" EXIT INT QUIT PIPE
 
 # start ws and browser in a detached network namespace
 SCRIPT='
-set -eu
```

Comment:
Please keep this.

Reply:
This slipped in by accident.  I thought I had removed the cockpit-desktop changes (except the proxy thing).